src/main/java/mate/academy/hw02/Popcorn.java

```diff
@@ -0,0 +1,47 @@
+package mate.academy.hw02;
+
+public class Popcorn {
+    void doSomthing() {
+        System.out.println("popcorn");
+    }
+    void secondMethod() {
```

Comment:
Popcorn  secondMethod, are you sure this is valid behavior for popcorn ?

Reply:
This is example local class